lib/tasks/factory_girl.rake

```diff
@@ -0,0 +1,15 @@
+namespace :factory_girl do
```

Comment:
@colinxfleming Automatically running the linter usually frowned upon because DB whacking can get out of hand as factory/models grow in complexity.

I'd recommend having the CI run the linter once as part of the build before `rake test` is invoked and recommend people run the linter rake task only when they make any changes to the FactoryGirl factories.

Something Fancy like Guard could automate this but that's probably too much overhead for now.